src/lib.rs

```diff
@@ -708,7 +709,7 @@ where
             network_direct_task_handle,
             committee_network_broadcast_task_handle: None,
             committee_network_direct_task_handle: None,
```

Comment:
These four network task handles are also `nll_todo()`, similar to `consensus_task_handle`.